onnxruntime/test/testdata/optimizers/test_adam.py

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think testdata should be for test _data_ that is committed. If this script _generates_ data for examination, should it be in `tools`?

Reply:
It is to generate test data for adamw_test, which might be helpful for maintaining the test cases. I did not find a good place in tools... still feel this might be a better place...